js/main.js

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function() {
+  $.ajax({
+      url: 'http://localhost:3000/articles',
+      type: 'GET'
+    }).done(ArticleApp.getArticles);
+  });
+
+      // $('#get-articles').click(ArticleApp.getArticles);
+
+      // $("input[value='Create Article']").click(function() {
+      //     event.preventDefault();
+      // });
+
+    var ArticleApp = ArticleApp || {};
+
+    ArticleApp.getArticles = function(data) {
+      var articles = data;
+      console.log(articles);
+      var $list = $('#articles');
+      for(var i = 0; i < articles.length; i++) {
+       var $newArticle = $('<li id="article-' + articles[i].id + '">');
+      if(articles[i].comments.length === 0){
+        $newArticle.html('<h3>' + articles[i].title + '</h3>' + ' <p>' + articles[i].body + '</p>');
```

Comment:
This is good. We'll see smoother ways of doing this soon, so if it feels clunky... it is!